docs/roles/integrator/exchange-integration.md

```diff
@@ -773,9 +771,993 @@ and we can find its outcome in `receipts_outcome`:
+
+In order to send FT to the account it should deposit a storage for it. To check if account has deposited the storrage for this FT do the following:
+
+Get storage balance of the account. `storage_balance_of` function returns the amount of deposited storage or `null` if there is no deposit.
+  - using `near-cli`:
+
+    ```bash
+    near view <contract_account_id> storage_balance_of '{"account_id": "<user_account_id>"}'
+    ```
+
+    Result:
+
+    ```bash
+    View call: ft.demo.testnet.storage_balance_of({"account_id": "serhii.testnet"})
+    null
+    ```
+
+  - with `JSON RPC` call:
+      ```bash
+      http post https://rpc.testnet.near.org jsonrpc=2.0 id=storagebalanceof method=query \
+      params:='{
+         "request_type": "call_function",
+         "finality": "final",
+         "account_id": "ft.demo.testnet",
+         "method_name": "storage_balance_of",
+         "eyJhY2NvdW50X2lkIjogInNlcmhpaS50ZXN0bmV0In0K"
+      }'
+      ```
+
+      Example responce:
+
+      ```bash
+      HTTP/1.1 200 OK
+      Alt-Svc: clear
+      Via: 1.1 google
+      access-control-allow-origin: 
+      content-length: 173
+      content-type: application/json
+      date: Wed, 02 Jun 2021 14:22:01 GMT
+      {
+          "id": "storagebalanceof",
+          "jsonrpc": "2.0",
+          "result": {
+              "block_hash": "EkM2j4yxRVoQ1TCqF2KUb7J4w5G1VsWtMLiycq6k3f53",
+              "block_height": 50054247,
+              "logs": [],
+              "result": [
+                  110,
+                  117,
+                  108,
+                  108
+              ]
+          }
+      }
+      ```
+
+      Decoded result in this case is `null`.
+
+Get the minimum storage required for FT
+  - using `near-cli`:
+
+    ```bash
+    near view <contract_account_id> storage_balance_bounds`
+    ```
+
+    Result:
+
+    ```bash
+    View call: ft.demo.testnet.storage_balance_bounds()
+    { min: '1250000000000000000000', max: '1250000000000000000000' }
+    ```
+
+  - with `JSON RPC` call
+      ```bash
+      http post https://rpc.testnet.near.org jsonrpc=2.0 id=storagebalancebounds method=query \
+      params:='{
+          "request_type": "call_function",
+          "finality": "final",
+          "account_id": "<contract_account_id>",
+          "method_name": "storage_balance_bounds",
+          "args_base64": ""
+      }'
+      ```
+
+      Example responce:
+      ```bash
+      HTTP/1.1 200 OK
+      Alt-Svc: clear
+      Via: 1.1 google
+      access-control-allow-origin: 
+      content-length: 357
+      content-type: application/json
+      date: Wed, 02 Jun 2021 15:42:49 GMT
+      
+      {
+          "id": "storagebalancebounds",
+          "jsonrpc": "2.0",
+          "result": {
+              "block_hash": "Fy3mBqwj5nvUDha3X7G61kmUeituHASEX12oCASrChEE",
+              "block_height": 50060878,
+              "logs": [],
+              "result": [
+                  123,
```

Comment:
I think just for readability, let's have all these bytes inline. so like

```
123, 34, 109…
```

instead of this really tall text